instance/models/mixins/openedx_database.py

```diff
@@ -70,7 +70,7 @@ def mysql_database_name(self):
         """
         The mysql database name for this instance
         """
-        return self.database_name
+        return self.generate_database_name()
```

Comment:
The only place we call `generate_database_name()` should be in OpenEdXInstance's `save()` method where we initially set the value of the `database_name` field. Everywhere else, we should keep referencing `self.database_name`.

That will ensure nothing will change for existing instances after we change the details of `generate_database_name` in the follow-up PR.

Reply:
Oh yes! My bad.